### PR TITLE
Review Branch

### PR DESCRIPTION
**Pull request by LuisSalas94**

### CODE REVIEW

File: modules/createTodo.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	const array = JSON.parse(localStorage.getItem("to-do"));
+
+	for (let i = 0; i < tasklist.length; i += 1) {
+		if (tasklist[i] === input) {
+			if (array[i].completed === false) {
+				array[i].completed = true;
+			} else {
+				array[i].completed = false;
+			}
+		}
```
Comment (franciscoPonceDev) [inline modules/createTodo.js:14]: Maybe you could use a for each loop and the code will be more readable

File: modules/createTodo.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+		input.name = "checkbox";
+		input.addEventListener("change", () => {
+			changeCompleted(input.parentElement);
+		});
+		if (todo.completed === true) {
+			input.setAttribute("checked", true);
+			li.className = "line";
+		} else {
+			li.className = "";
+		}
```
Comment (franciscoPonceDev) [inline modules/createTodo.js:69]: I Saw that you marked this code in an issue. I'll just mark it here so you can work on that in this exercise.

File: modules/createTodo.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+	localStorage.setItem("to-do", JSON.stringify(array));
+	/* eslint-disable-next-line */
+	createTodo(array);
+}
+
+export default function createTodo(todos) {
+	const todosBody = document.querySelector(".todos-body");
+	todosBody.innerHTML = "";
+
+	const todoUl = document.createElement("ul");
```
Comment (franciscoPonceDev) [inline modules/createTodo.js:40]: I Saw that you marked this code in an issue. I'll just mark it here so you can work on that in this exercise.

**Review by franciscoPonceDev — COMMENTED** [review]
## exercise suggestions

Your code is realy good! Just some few changes to make it even better.

File: modules/createTodo.js (hunk 3 above)
Comment (LuisSalas94) [reply modules/createTodo.js:40]: Thanks for the feedback.

File: modules/createTodo.js (hunk 2 above)
Comment (LuisSalas94) [reply modules/createTodo.js:69]: Thanks for the feedback.

File: modules/createTodo.js (hunk 1 above)
Comment (LuisSalas94) [reply modules/createTodo.js:14]: I will take that into consideration.Thanks

**Pull request merged**